shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/navigation/KotlinConfNavHost.kt

```diff
@@ -43,10 +45,24 @@ import org.jetbrains.kotlinconf.screens.TermsOfUse
 import org.jetbrains.kotlinconf.utils.getStoreUrl
 import kotlin.reflect.typeOf
 
+fun navigateToSession(notificationId: String) {
+    val sessionId = SessionId(notificationId.substringBefore("-"))
+    sessionIds.trySend(sessionId)
+}
```

Comment:
We can make this more sophisticated later, especially if we need to link to other types of pages, like news